Double Spacing in Paragraph Text solved

RonaldRonnie opened the pull request:
The double spacing has been removed to improve efficience and readability

megha070 commented:
Only six colors are there in feedback, I have included all.
![image](https://user-images.githubusercontent.com/40516822/77677481-c69a8300-6fb5-11ea-9d20-a2f34a31979c.png)

RonaldRonnie committed:
Double Spacing in Paragraph Text solved

RonaldRonnie committed:
feat: Improve toggle functionality with cleaner
code structure and reusable functions

- Refactored button and panel toggle logic
 into a reusable utility function
- Added helper function for element selection
to enhance readability
- Applied conditional event listener assignments
 using short-circuiting (&&)
- Ensured consistent behavior for text updates

 based on button state
- Maintained original functionality while
adopting a cleaner and modern coding style

Queen-codes commented:
Hi @RonaldRonnie , I noticed that your first commit was aimed at solving #248, but the issue hasn’t been labeled as ready-for-work yet. Please read the [Contribution Guidelines](https://opensource.creativecommons.org/contributing-code/) for more information on how the contribution works in this community.

It seems your second commit addresses an issue that hasn’t been raised yet. Generally, it's best to avoid submitting PRs for unraised issues. If you identify a problem, feel free to create an issue first. Once mentors review it, they’ll decide whether to move forward, and you can then create a PR.

I’ve also left a [comment](https://github.com/creativecommons/vocabulary/pull/255#issuecomment-2421449526) on one of your PRs regarding formatting, which applies here as well.

possumbilities commented:
Hi @RonaldRonnie Thanks for your enthusiasm!

Please do not open PRs that are not attached to approved Issues. I'm closing this for now, feel free to writeup concise Issues for the above, and wait until they move to https://github.com/creativecommons/vocabulary/labels/%F0%9F%8F%81%20status%3A%20ready%20for%20work before proceeding with associated PRs.

Please see the [Contribution Guidelines](https://opensource.creativecommons.org/contributing-code/) for more information.

Closing for now.
